ARCs/arc-0071.md

```diff
@@ -0,0 +1,219 @@
+---
+arc: 71
+title: Conventions for Soul Bound ASAs
+description: Algorand Standard Asset Parameters Conventions for Soul Bound Token
+author: Jupiter Yu (@JupiterXiaoxiaoYu)
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Draft
+type: Standards Track
+category: ARC
+created: 2022-08-14
+---
+
+# Community Algorand Standard Asset Parameters Conventions for Soul Bound Token
+
+
+
+## Abstract
+
+We introduce Algorand Standard Asset Parameters Conventions for Soul Bound Token [ARC-71](./arc-0071.md), this is a specific application of [ARC-69](./arc-0069.md).
```

Comment:
yeah, you are right, let's figure out how it can be compatible with those. My first thought was send a asset configuration transaction to turn those into soul-bound.